apps/omg_child_chain/lib/omg_child_chain/fees/file_adapter.ex

```diff
@@ -69,7 +69,14 @@ defmodule OMG.ChildChain.Fees.FileAdapter do
     end
   end
 
+  # Get the specs file path from the provided opts. If not configured, it defaults back to
+  # the child_chain source code's priv/fee_specs.json.
+  #
+  # Beware that we check with `is_binary/1` because a nil path should also use the default.
   defp get_path(opts) do
-    "#{:code.priv_dir(:omg_child_chain)}/#{Keyword.fetch!(opts, :specs_file_name)}"
+    case Keyword.fetch(opts, :specs_file_path) do
+      {:ok, path} when is_binary(path) -> path
+      _ -> Application.app_dir(:omg_child_chain, "priv/fee_specs.json")
```

Comment:
in the other place we're using `:code.priv_dir(:omg_child_chain)}`. Can we unify? I'm opting for `priv_dir`